app/assets/javascripts/ajax_mixpanel_events.js

```diff
@@ -14,7 +14,7 @@ var ajaxMixpanelEvents = (function () {
         var attributePrefix = "trackAttribute";
         if (key.indexOf(attributePrefix) === 0) {
           var propertyKey = key.substring(attributePrefix.length).toLowerCase();
-          eventData.append("event["+propertyKey+"]", value);
+          eventData.append("event[data]["+propertyKey+"]", value);
```

Comment:
there are a couple specific properties that are pulled directly out of the event dictionary but new custom properties like these will only be attached if they are in the data dictionary under it.

Reply:
@loumoore  i'm still not totally clear on what effect this has - does it fix the tracking we're currently doing with `data-track-click`? is it related to this PR or just something you noticed?

Reply:
both :) I wanted to add click tracking to the links on the stimulus page so we know how many people we are sending to the IRS websites especially, and in testing those I found that the data-track-attribute-* elements on existing click tracking were not being sent to mixpanel. So it's a bug fix I stumbled upon in the course of building the stimulus page.